### ISSUE TITLE
Error when launching vms.

### ISSUE THREAD

**Issue by kshlm** [pradels/vagrant-libvirt#292]
I get the following error when trying to launch any vm.
```
/home/kaushal/.vagrant.d/gems/gems/fog-core-1.27.3/lib/fog/formatador.rb:6:in `initialize': wrong number of arguments (0 for 1) (ArgumentError)
        from /home/kaushal/.vagrant.d/gems/gems/fog-core-1.27.3/lib/fog/core/model.rb:24:in `new'
        from /home/kaushal/.vagrant.d/gems/gems/fog-core-1.27.3/lib/fog/core/model.rb:24:in `inspect'
        from /home/kaushal/.vagrant.d/gems/gems/fog-core-1.27.3/lib/fog/core/collection.rb:19:in `inspect'
        from /home/kaushal/.vagrant.d/gems/gems/fog-core-1.27.3/lib/fog/core/collection.rb:19:in `to_s'
        from /home/kaushal/.vagrant.d/gems/gems/vagrant-libvirt-0.0.24/lib/vagrant-libvirt/action/set_name_of_domain.rb:17:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/warden.rb:34:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/warden.rb:34:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/warden.rb:34:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/builder.rb:116:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/runner.rb:66:in `block in run'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/util/busy.rb:19:in `busy'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/runner.rb:66:in `run'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/builtin/call.rb:53:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/warden.rb:34:in `call'
        from /home/kaushal/.vagrant.d/gems/gems/vagrant-libvirt-0.0.24/lib/vagrant-libvirt/action/connect_libvirt.rb:18:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/warden.rb:34:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/builtin/config_validate.rb:25:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/warden.rb:34:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/builder.rb:116:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/runner.rb:66:in `block in run'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/util/busy.rb:19:in `busy'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/action/runner.rb:66:in `run'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/machine.rb:212:in `action_raw'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/machine.rb:189:in `block in action'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/environment.rb:516:in `lock'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/machine.rb:176:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/machine.rb:176:in `action'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.1/lib/vagrant/batch_action.rb:82:in `block (2 levels) in run'
```
(full debug logs are present at https://gist.github.com/kshlm/3037aacd6a1ed6344590)

This just started happening all of a sudden.

System information:
Archlinux with 3.18.2 kernel
libvirt-1.2.11
vagrant-1.7.2
vagrant-libvirt-0.0.24
ruby-2.2.0

**Comment by sciurus** [comment]
The error is coming from fog-core

They try to [create](https://github.com/fog/fog-core/blob/v1.27.3/lib/fog/core/model.rb#L24) a formattador object without the [argument](https://github.com/fog/fog-core/blob/v1.27.3/lib/fog/formatador.rb#L6) it requires.

Could you open a bug with fog-core? Seems like we'll want to lock an older version until this is fixed.

**Comment by kshlm** [comment]
@sciurus I've opened an issue on fog-core.
fog-core was updated 4 days back to a new version. I had done a plugin uninstall and re-install, which seems to have pulled in the newer fog core, instead of the one I had earlier.

Is there any way I can continue working in the meantime till the issue is fixed? Would replacing the contents of the gem directory with the old version help?

**Comment by kshlm** [comment]
I installed fog-core-1.27.2 as a vagrant plugin `vagrant plugin install fog-core --plugin-version=1.27.2` and I got vagrant-libvirt to work again.

**Comment by kshlm** [comment]
Upstream fog-core fix was merged https://github.com/fog/fog-core/pull/120 . Hence closing this.

**Comment by sciurus** [comment]
This should be fixed in fog-core 1.27.4 which was just released.
